daffodil-core/src/main/scala/org/apache/daffodil/dpath/DFDLExpressionParser.scala

```diff
@@ -17,8 +17,8 @@
 
 package org.apache.daffodil.dpath
 
-import java.math.{ BigDecimal => JBigDecimal }
-import java.math.{ BigInteger => JBigInt }
+import java.math.{BigDecimal => JBigDecimal}
+import java.math.{BigInteger => JBigInt}
```

Comment:
Our Daffodil wiki's [Intellij IDEA Setup](https://cwiki.apache.org/confluence/display/DAFFODIL/IntelliJ+IDEA+Setup) has the following comment by Mike:

> Some of IDEA's Scala settings are not consistent with our formatting standard. On the Spaces tab, check boxes for "Import braces".

Therefore, is there a way to stop scalafmt from removing these whitespace characters from import braces?

Reply:
Yes, that would be `spaces.inImportCurlyBraces = true` and I have added that to the config.